packages/react-hooks/src/inbox/use-inbox.ts

```diff
@@ -27,15 +31,15 @@ const useInbox = () => {
           return;
         }
 
-        actions.newMessage(courierEvent?.data);
+        actions.newMessage(courierEvent?.data as ICourierMessage);
       },
     });
 
     transport?.listen({
       id: "event-listener",
       type: "event",
       listener: (courierEvent) => {
-        const data = courierEvent?.data;
+        const data = courierEvent?.data as ICourierEventMessage;
```

Comment:
Do we need any validation before we cast this?

Reply:
the type "event" means the type should be ICourierEventMessage.  its possible we have different listener functions that already have the type case so transport.eventListener